students/Amit/Session02/series.py

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env python
+
+fib_cache= {}
+
+def fibonacci(n):
+    if n in fib_cache:
```

Comment:
very slick! This is called "memoization", and we'll get into a better way to do this in a few weeks.

but nice, and should really enhance performance, not computing the same one multiple times.